pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -131,6 +131,9 @@ func init() {
 		authorizationapi.ClusterRole{
 			ObjectMeta: kapi.ObjectMeta{
 				Name: BuildControllerRoleName,
```

Comment:
add these annotations in addServiceAccount? aren't all service account roles infra only?